tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -29,3 +30,17 @@ def test_event_invoices_mark_due(self):
 
             self.assertEqual(status_new, "due")
             self.assertNotEqual(status_paid, "due")
+
+    def test_delete_ticket_holders_with_no_order_id(self):
+        """Method to test deleting ticket holders with no order id after expiry time"""
+        with app.test_request_context():
+            ticket_holder_new = TicketHolder(firstname='John', lastname='Doe')
+
+            db.session.commit()
+
+            ticket_holder_id = ticket_holder_new.id
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ticket_holder_id' is assigned to but never used](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4674573)